movies-manager/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "movies-manager",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start:dev": "cross-env NODE_ENV=development webpack-dev-server",
+    "build:dev": "cross-env NODE_ENV=development webpack",
+    "start:prod": "cross-env NODE_ENV=production webpack-dev-server",
+    "build:prod": "cross-env NODE_ENV=production webpack",
+    "stat": "webpack --json --profile >stats.json",
```

Comment:
Also curious how and for what you will use stat command

Reply:
stat will form jason, that I can view on github analyze

Reply:
ok